app/assets/stylesheets/application.css

```diff
@@ -1,18 +1,89 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
- h1 {
+h1 {
   text-align: center;
 }
+/* @import url('https://fonts.googleapis.com/css?family=Numans'); */
+
+html,body{
```

Comment:
add space after comma and body i.e., `html, body {`
Also update on other places